mantidproject/mantid
# Update DirectILL documentation

**Description of work.**

This PR updates the ILL's direct geometry reduction documentation to use real experimental data.

**Report to:**[nobody].

**To test:**

Build the `docs-html` target and check that the usage examples in the `DirectILL*` algorithm pages and the code samples in the DirectILL technique page look OK.

Fixes #20453.

*This does not require release notes* because **this is a documentation update, no functional changes included**.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
